### ISSUE TITLE
Node 10.10 & 10.11 potential issues npm doesnt exist- node.exe vs node64.exe

### ISSUE THREAD

**Issue by JoshRosenstein** [nvm-windows/nvm#398]
### My Environment

- [ ] Windows 7 or below (not truly supported due to EOL - see wiki for details)
- [ ] Windows 8
- [ ] Windows 8.1
- [X] Windows 10
- [ ] Windows 10 IoT Core
- [ ] Windows Server 2012
- [ ] Windows Server 2012 R2
- [ ] Windows Server 2016

- [ ] My Windows installation is non-English.

### I'm using NVM4W version:

- [X ] 1.1.7
- [ ] 1.1.6
- [ ] 1.1.5
- [ ] 1.1.4
- [ ] 1.1.3
- [ ] 1.1.2
- [ ] 1.1.1
- [ ] Older
- [ ] OTHER (Please Specify)

### I have already...

- [X] read the README to be aware of npm gotchas & antivirus issues.
- [X] reviewed the [wiki](https://github.com/coreybutler/nvm-windows/wiki) to make sure my issue hasn't already been resolved.
- [X] verified I'm using an account with administrative privileges.
- [X] searched the issues (open and closed) to make sure this isn't a duplicate.
- [X] made sure this isn't a question about how to use NVM for Windows, since [gitter](https://gitter.im/coreybutler/nvm-windows?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) is used for questions and comments.

### My issue is related to (check only those which apply):

- [ ] settings.txt
- [ ] proxy support (Have you tried version 1.1.0+?)
- [ ] 32 or 64 bit support (Have you tried version 1.1.3+?)
- [ ] Character escaping (Have you tried version 1.1.6+?)
- [ ] A standard shell environment (terminal/powershell)
- [ ] A non-standard shell environment (Cmder, Hyper, Cygwin, git)

### Expected Behavior

Not sure if this is a nvm or Node issue, I noticed this issue when downloading node 10.10.0 and  node 10.11.0. did not have this issue with 10.9.0 or below.
but  `npm.cmd` tries the call node.exe but the exe file name is node64.exe. So either had to rename the node exe file, or update the `npm.cmd` line.

```sh
SET "NODE_EXE=%~dp0\node.exe"
```
to
```sh
SET "NODE_EXE=%~dp0\node64.exe"
```

just a heads up if anyone gets a cmd error of npm doesnt exist.

**Comment by coreybutler** [comment]
Was this a new install? NVM4W rewrites node64.exe to node.exe for you, so this shouldn't be an issue. This can be solved with `nvm use 10.10.0 64`, but I'd like to know more about the setup to see if we an figure out why the rename didn't occur.

**Comment by JoshRosenstein** [comment]
Yes, it was a new installs, come to think of it, i might have ran `nvm install` and `nvm use` prior to fixing the other issue of changing path variables to short (non-spaced).That may have caused nvm not to properly locate and make the change.
I ran the following, without the arch option.
```sh
nvm install 10.10.0
nvm use 10.10.0
```
Which led to having node64.exe within the folder.

**Comment by coreybutler** [comment]
It sounds like the environment may be a little unusual and is the most likely cause of the issue. However; I think this is a good edge case for illustrating the need for validation. I'm going to mark this as an enhancement request.

Note to self: NVM4W should check for alternative architecture and at least warn the user if node.exe does not exist and either node32.exe or node64.exe exist. When possible, auto-correct.

**Comment by martinstefanov** [comment]
I have had the same problem on Win 7 Pro with nvm 1.1.6. nvm was reporting that cannot download npm. I found that the npm repository was moved. I have upgraded to nvm 1.1.7 and the problem was fixed.

**Comment by agrcrobles** [comment]
I wasn't able to download the npm neither because it was moved from https://github.com/npm/cli/archive/.
I am on 1.1.7 and Windows 10 Node 8.

**Comment by pingoo** [comment]
Same here.
Windows 10 pro 64bits.

**Comment by Zougi** [comment]
Fix is in version 1.1.7 https://github.com/coreybutler/nvm-windows/blob/1.1.7/src/nvm/web/web.go#L21
Thank you @coreybutler

**Comment by michaelmberger** [comment]
Ran into same issue on 1.1.7. Opted to hard link node.exe to node64.exe rather than edit the script. 
Possible other causes: I moved the installation directory from Appdata/Roaming/nvm to c:\Program Files\nodejs\ then did `nvm install latest`

**Comment by coreybutler** [comment]
Closing due to age.
